src/ubuntu/18.04/crossdeps/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-coredeps
+
+# Install the base toolchain we need to build anything (clang, cmake, make and the like).
+RUN apt-get update \
+    && apt-get install -y \
+        apt-transport-https \
+        ca-certificates \
+        gnupg \
+        software-properties-common \
+        wget \
+    && wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc | apt-key add - \
+    && apt-add-repository 'deb https://apt.kitware.com/ubuntu/ xenial main' \
```

Comment:
```suggestion
    && apt-add-repository 'deb https://apt.kitware.com/ubuntu/ bionic main' \
```

Reply:
We should not change to `bionic` in the src/ubuntu/18.04/cross/arm64/hooks/pre-build. We want to build rootfs for 16.04 so that the binaries we produce in crossbuild can be used on 16.04 too. Its end of standard support is April 2021.

Reply:
@janvorli i am going to follow the existing style and target 18.04. We may actually need these in the future. My plan is to add a 16.04 targeting folder, similar to how we used to target 14.04 from the 16.04 docker containers in the past. I plan to do this in a follow up pr though